state-chain/pallets/cf-ingress-egress/src/migrations/add_boost_pools.rs

```diff
@@ -50,6 +69,8 @@ impl<T: Config<I>, I: 'static> OnRuntimeUpgrade for Migration<T, I> {
 			},
 		);
 
+		let _ = old::PrewitnessedDeposits::<T, I>::clear(u32::MAX, None);
```

Comment:
this will have to be in another migration, since this migration will be applied in 1.4 - unless we cherry pick into 1.4. But think I'd rather not, and we try to focus on stabilising the release now with only essential commits. So yeah, will need a separate migration, and pallet version bump etc.

Reply:
Done. I also bumped the spec version as discussed.